cdk/kittyhawk/package.json

```diff
@@ -65,7 +65,7 @@
   "main": "lib/index.js",
   "license": "MIT",
   "homepage": "https://kittyhawk.pennlabs.org",
-  "version": "1.1.2",
+  "version": "0.0.0",
```

Comment:
Version number is reset. We need to change this back to reflect the `1.1.2` version

Reply:
Addressed